RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,221 @@
+be determined by the Council and can be increased or decreased at the Council’s discretion. We anticipate Council members will spend less than 5
+hours a month on Council activities.
+
+Portions of the sessions will be recorded and published to YouTube. Discussion topics under a Non-Disclosure Agreement (NDA) will be redacted from
+the recording before it is published.
+
+### The Council meetings include two sections
+
+(1) Town hall
+
+* The AWS CDK team will report on project release priorities and the status of AWS-led features in each Council meeting. Council members will have
+an opportunity to ask questions and deliver feedback.
+* Community representatives will report on news and share updates on community-led features. Council members will have an opportunity to ask
+questions and deliver feedback.
+
+(2) New proposals
+
+* The Council will review and debate RFCs/features, feature docs, or new processes brought forth by the CDK community and AWS. AWS may not be able
+to report on all AWS-led projects when we are bound by legal requirements, customer agreements, or when it would put AWS at a competitive
+disadvantage to report on some project.
+* Anyone in the CDK community may propose topics for discussion by the Council. The agenda is finalized one week before each meeting to allow time
+for pre-reading and preparation. Proposers are invited to the meeting to present and advocate for their change. If they are unable to attend,
+their attendance is not required for the Council to make a decision.
+* The Council discusses the change’s appropriateness, merits, scope of work, and whether the change can be community-led or if it will be built by
+internal Amazon resources.
+* Decisions are completed and recorded through the following mechanism:
+  * Each Council member enters an opinion on whether the feature is viable including a short explanation. The opinions and reasons behind them are
+recorded on the change proposal. Opinion submissions follow the [Chatham House Rule](https://www.ibabs.com/en/glossary/chatham-house-rule/#:~:text=The%20Chatham%20House%20Rule%20reads,')
+by default, though Council members may choose to sign them.
+  * If the change is deemed appropriate for inclusion in CDK by the Council, the AWS CDK Product team decides whether the feature will be built by
+AWS and adds it to their feature backlog, or if the feature will be offered to the community for implementation.
+  * If a change is deemed to be appropriate for community-led development, the label 'open-for-community-contribution' will be added to the feature
+request.
+  * It is preferred that community-led features are code-reviewed by a
+[Distinguished Contributor](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#badges-pilot-program) before AWS reviews them.
+    * If a Distinguished Contributor is not available within 30 days, AWS will review.
```

Comment:
Looks like a single digit number of contributors reached receiving this [contribution level label](https://github.com/aws/aws-cdk/labels/distinguished-contributor) without working for Amazon? Worth considering including [community reviewers](https://github.com/aws/aws-cdk/wiki/CDK-Community-PR-Reviews#trusted-cdk-reviewers) and knocking the [contribution badge level](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#badges-pilot-program) and waiting period down a notch so community PRs aren't waiting on a small group of external contributors for a month